### PR TITLE
Added Default favicon

### PR DESCRIPTION
**Pull request by ddomdel**
I added a default favicon in the static folder. Let me know if this is the right one or if I added it correctly. Thanks!

This is related to this issue: https://github.com/CTFd/core-beta/issues/8

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <title>{{ Configs.ctf_name }}</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="{{ url_for('views.themes', theme='core', path='img/favicon.ico') }}" type="image/x-icon">
```
Comment (ColdHeat) [inline templates/base.html:7]: i think the endpoint should be `views.themes_beta` for the core-beta theme for now. Also we should probably be just using the core-beta assets so no need to specify the theme. If you need to you can copy over the favicon file into the assets folder and then modify vite.config.js to copy that into the static folder.

Comment (ddomdel) [reply templates/base.html:7]: Alright. I modified it now. 

I have a question. I am running this theme inside a local docker container and when you open the site in the browser and check where it got the favicon.ico, it says that it got it from core-beta/static/img folder
image
But, when I check the files in VSCode it doesn't have the img folder inside the static folder.
image

**Comment by ddomdel** [comment]
- Added `img `folder to `assets` folder. 
- Configured Vite to copy the contents of `img` folder to `static/assets` folder.
- Modified `<Link/>` href to look into `/static/assets` folder.
assets-location
Verified if it is really looking into the intended folder to look into, which should be `/themes/core-beta/static/assets/favicon.ico`, by inserting a JS script in `base.html` and letting flask-jinja render it
js-test-asset-location
assets-location1

**Comment by ColdHeat** [comment]
The static files I don't think should have been deleted from here. Can you put them back?

**Comment by ddomdel** [comment]
Yeah I noticed that too. But I think it happens when vite recreates those static files, it creates a new hash on its name. So git thinks that it has been deleted I think. Not really sure, but I'll restore it instead.

File: vite.config.js [hunk 2]
```diff
@@ -32,6 +32,10 @@ export default defineConfig({
               src: "./node_modules/@fontsource/raleway/files/**/*400*",
               dest: "static/webfonts",
             },
+            {
+               src: "./assets/img/**",
+               dest: "static/assets",
```
Comment (ColdHeat) [inline vite.config.js:37]: This should be `static/img`

**Comment by ColdHeat** [comment]
This isn't right because you can't override it from the Admin Panel. 

See https://github.com/CTFd/CTFd/blob/master/CTFd/themes/core/templates/base.html#L7.

**Pull request merged**